docs/guide/understanding/task/index.md

```diff
@@ -75,7 +75,7 @@ let theTask = Task.tryOrElse(
 let resolved = Task.resolve(123);
 
 // `rejected` has the type `Task<never, string>`
-let rejected = Task.rejecte("something went wrong");
+let rejected = Task.rejected("something went wrong");
```

Comment:
Ah, good catch, but it actually needs to be `reject` to match `Promise`. Thank you!

```suggestion
let rejected = Task.reject("something went wrong");
```